workloads.py

```diff
@@ -160,4 +160,12 @@
         .wait_for_service_existence("yb-tserver-ui", common.EXIST)
         .wait(70),
     },
+    "nifikop-operator": {
+        "change-config": test_framework.new_built_in_workload()
+        .cmd("kubectl apply -f test-nifikop-operator/test/nc.yaml")
+        .wait(10).wait_for_pod_status("simplenifi-1-*", common.RUNNING)
```

Comment:
Why do we use hardcoded wait before wait_for ?

Reply:
This `wait` is useless, I can get rid of it. There is another `wait` after apply the config change, that wait is to wait operator to delete the pod (that time can be relatively short), then wait for RUNNING.